src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -143,16 +151,60 @@ function DisplayLine({
   )
 }
 
+function DollarLineAndAtLine({
+  line,
+  container = 'div'
+}: {
+  line: LineContainer
+  container?: string
+}): JSX.Element {
+  if (line.type === 'RulingDollarLine') {
+    const rulingDollarLine = line as RulingDollarLine
+    return React.createElement(
+      container,
+      { className: classNames([`Transliteration__ControlLine`]) },
```

Comment:
Please use the correct element name. This is `RulingDollarLine` not `ControlLine`.